srcpkgs/png++-devel/template

```diff
@@ -0,0 +1,30 @@
+# Template file for 'png++-devel'
+pkgname=png++-devel
+version=0.2.9
+revision=1
+noarch=yes
+_pkgname=png++
+build_style=gnu-makefile
+make_build_target="docs"
```

Comment:
This **might** make it not build the main target, make sure it does build the main target

Reply:
The main target is an example and is not needed nor packaged.